### PR TITLE
add release note

### PR DESCRIPTION
**Pull request by Binyang2014**

### CODE REVIEW

**Comment by hzy46** [comment]
Some other files should be changed too. Please refer to : https://github.com/microsoft/pai/wiki/How-to-Create-a-PR-for-New-Release

**Review by debuggy — APPROVED** [review]

File: RELEASE_NOTE.md [hunk 1]
```diff
@@ -1,5 +1,32 @@
...
+
+- Alert system enhancement
+  - Add alert & auto-fix for GPU perf issue #5342 #5383
+  - Add `kill-long-running-job` email templates #5384
+  - Add alert for API server cert expired #5334
+
+- Support sort by `completionTime` for get job list API #5347
+
+- Deployment
+  - Support add/remove nodes #5321 #5167. (**Warning** `config.yaml` need to be added for this feature. refer: https://github.com/microsoft/pai/blob/master/docs/manual/cluster-admin/how-to-add-and-remove-nodes.md#pull--modify-cluster-settings)
```
Comment (suiguoxin) [inline RELEASE_NOTE.md:25]: Support add/remove nodes with `paictl.py`

File: RELEASE_NOTE.md [hunk 2]
```diff
@@ -1,5 +1,32 @@
...
+  - New submission page replaces `Advanced` with `More info` and places it under each section to improve user experience.
+  - In new submission page, the sidebar can be shrank to give the main area more visual space.
+  - The new submission page moves the yaml editor into a single page, which allows user to focus on setting config or editing yaml protocol.
+  - The new submission page improves the responsive design in small and medium resolution.
+  
+> Know Issue: Tensorboard tool is not implemented in the new submission page yet. If you need to use it, please use the old version.
+
+- Alert system enhancement
+  - Add alert & auto-fix for GPU perf issue #5342 #5383
+  - Add `kill-long-running-job` email templates #5384
```
Comment (suiguoxin) [inline RELEASE_NOTE.md:19]: Finally we didn't add new templates, suggest to change to `refine kill-low-efficiency-job-alert email templates`

File: RELEASE_NOTE.md (hunk 1 above)
Comment (Binyang2014) [reply RELEASE_NOTE.md]: Fixed

File: RELEASE_NOTE.md (hunk 2 above)
Comment (Binyang2014) [reply RELEASE_NOTE.md]: Fixed

**Review by suiguoxin — APPROVED** [review]

**Pull request merged**